### PR TITLE
[feature] Travis CI and Actions CI

### PR DESCRIPTION
**Pull request by sourTaste000**
kek

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,6 +5,8 @@
 [![Paypal](https://img.shields.io/badge/paypal-donate-red?color=169bd7&logo=paypal&style=flat-square)](https://paypal.me/mik4a/5USD)
 [![BTC](https://img.shields.io/badge/btc-donate-red?color=f08b16&logo=bitcoin&style=flat-square)](https://www.blockchain.com/btc/address/19pH4aNZZMPJkqQ2826BauRokyBs1NYon7)
 [![BCH](https://img.shields.io/badge/bch-donate-red?color=2db300&logo=cash-app&style=flat-square)](https://www.blockchain.com/bch/address/19pH4aNZZMPJkqQ2826BauRokyBs1NYon7) 
+[![Build Status](https://travis-ci.com/kami-blue/bot-kt.svg?branch=master)](https://travis-ci.com/kami-blue/bot-kt) 
+[![CodeFactor](https://www.codefactor.io/repository/github/kami-blue/bot-kt/badge)](https://www.codefactor.io/repository/github/kami-blue/bot-kt)
```
Comment (l1ving) [inline README.md:9]: Put these before donate. Also all the badges should use the same style

**Review by l1ving — CHANGES_REQUESTED** [review]
Wrapper jar does not need to be included

**Comment by sourTaste000** [comment]
wrapper jar has to be included or it will give this error:
```
Error: Could not find or load main class org.gradle.wrapper.GradleWrapperMain
```

**Comment by l1ving** [comment]
Stupid travis. It's supposed to download the jar with the wrapper

**Comment by sourTaste000** [comment]
😂

**Comment by wnuke** [comment]
#36 Adds CI already, so this is probably not needed.

**Comment by sourTaste000** [comment]
i added travis

**Comment by wnuke** [comment]
That's what I mean, #36 adds a workflow for GitHub's built-in CI, it also publishes Docker containers for downloading. GitHub actions is exactly like Travis, except it's built-in to GitHub.

**Comment by sourTaste000** [comment]
why u bulli me

**Review by l1ving — CHANGES_REQUESTED** [review]
Please update this branch to latest master. I'll do it tomorrow morning if you haven't done it.

**Comment by sourTaste000** [comment]
my brain is too small to figure out how to fix this conflict XD
if you could do it, that would be great!
thanks

**Review by l1ving — APPROVED** [review]

**Pull request merged**